src/plugins/discover/public/application/view_alert/view_alert_route.tsx

```diff
@@ -69,6 +69,7 @@ export function ViewAlertRoute() {
       displayRuleChangedWarn,
       displayPossibleDocsDiffInfoAlert,
       showDataViewFetchError,
+      dataViewUpdatedWarn,
```

Comment:
nit: for consistency I'd suggest to label it:
`showDataViewUpdatedWarning`
since all other functions were labeled in the doSomethingNiceOrNasty was